### PR TITLE
Add BertTokenizer to iOS package ops config.

### PR DESCRIPTION
**Pull request by edgchen1**

### CODE REVIEW

**Review by YUNQIUGUO — DISMISSED** [review]

File: tools/ios/package_ops.config [hunk 1]
```diff
@@ -1 +1 @@
-com.microsoft.extensions;1;DecodeImage,EncodeImage
+com.microsoft.extensions;1;DecodeImage,EncodeImage,BertTokenizer
```
Comment (wenbingl) [inline tools/ios/package_ops.config:1]: is it in "ai.onnx.contrib" domain?

Comment (edgchen1) [reply tools/ios/package_ops.config:1]: hm, it is. the op config parsing doesn't seem to care about that now though.

**Review by wenbingl — APPROVED** [review]

File: tools/ios/package_ops.config (hunk 1 above)
Comment (skottmckay) [reply tools/ios/package_ops.config:1]: the op config parsing probably doesn't matter unless we had different implementations in the old and new domains (which I hope is unlikely). 

the model would care though, and ideally we use the new domain when updating models, so we'll need to also register the tokenizer in the new domain. 

maybe we could add something to only register ops in the new domain when building the mobile packages, but I don't think that would make much of a difference to binary size (would be slightly cleaner though).

Comment (edgchen1) [reply tools/ios/package_ops.config:1]: updated to also register the tokenizer ops in the new domain. so now the config should be correct. perhaps we can look into stricter validation in the op config parsing later.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**